common/app/views/fragments/containers/facia_cards/container.scala.html

```diff
@@ -43,11 +43,16 @@
                  data-component="@componentName"
                  aria-expanded="true">
 
-                @if(containerDefinition.isStoryPackage){
+                @if(containerDefinition.isStoryPackage && !experiments.ActiveExperiments.isParticipating(experiments.Garnett)){
                     <style>
                     @Html(common.Assets.css.inlineStoryPackage)
                     </style>
                 }
+                @if(containerDefinition.isStoryPackage && experiments.ActiveExperiments.isParticipating(experiments.Garnett)){
+                    <style>
+                    @Html(common.Assets.css.inlineStoryPackageGarnett)
```

Comment:
We could comment out this line instead of commenting out all the story package CSS

Reply:
This is to make sure that the new stye sheet gets loaded into the page, and it's therefor styled like a normal container...for now.